Bottom: 9dc26f9e75afe38a3a828b47fba818842deef383
Top:    59caab22b8e6a4a504cd10a443f256408ba241df
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 13:53:03 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/Vagrantfile b/Vagrantfile
index e037c694..6ed5da6d 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -40,11 +40,20 @@ Vagrant.configure("2") do |config|
         main.vm.hostname = "vmdaedalus.novatech-llc.com"
         main.vm.provider "virtualbox" do |vb|
             vb.name = "vmdaedalus"
-            vb.memory = 4096
+            vb.memory = 6000
             vb.cpus = 4
         end
     #    config.vm.network "public_network", ip: "192.168.0.100", netmask:"255.255.0.0", bridge: "enp0s31f6"
-        main.vm.network "private_network", ip: "192.168.0.100", netmask:"255.255.0.0"
+        main.vm.network "private_network", 
+            ip: "192.168.0.100", 
+            netmask:"255.255.0.0"
+
+        main.vm.network "forwarded_port",
+            guest: 2375,
+            host: 12375,
+            protocol: "tcp",
+            id: "docker"
+
         main.vm.synced_folder "/mnt/vm/bacula-restores",
             "/mnt/bacula-restores",
             id: "bacula-restores",
@@ -60,7 +69,11 @@ Vagrant.configure("2") do |config|
             vb.name = "bbworker"
             vb.cpus = 1
         end
-        worker.vm.network "private_network", ip: "192.168.0.150", netmask:"255.255.0.0"
+
+        worker.vm.network "private_network", 
+            ip: "192.168.0.150", 
+            netmask:"255.255.0.0"
+
         worker.vm.provision "shell", inline: $bbworker
 
     end
